src/recursionAssignment/Recursion.java

```diff
@@ -1,5 +1,50 @@
 package recursionAssignment;
 
-public class Recursion{
+public class Recursion {
 
+    /**
+    *	Counts how many times the digit 7 is present in the given number.
+    * 	@author: Joshua. S 
```

Comment:
include @param specifications for your javadoc comments to indicate parameters for your methods.